assets/scss/styles.scss

```diff
@@ -13,6 +13,7 @@
  */
 
```

Comment:
What's the reason behind all the styles changes ?

Reply:
Newer versions of stylelint standard config, have new rules, like when you upgrade php cs fixer and some code needs fixing.